### ISSUE TITLE
Improve copy experience

### ISSUE THREAD

**Issue by mfrisbey** [hlxsites/aem-assets-boilerplate#37]
In the demo, users check the box for an asset, then click "Select", after which the asset is copied to the clipboard to allow pasting into a document.

* We should at least explore providing some kind of feedback to the user that the asset has been copied to the clipboard. Currently nothing happens, which might lead to user confusion.
* Lower priority, but can we change the text of the button to something like "Copy to Clipboard" instead?
* It would be nice if we could reduce the number of clicks. Could we add a "Copy" button on hover? Or maybe the copy happens just by clicking an asset?

**Comment by sdmcraft** [comment]
Right now there's a proxy being used in the middle to fetch assets from AEM and make available in Franklin. I ended up adding it 
- for bypassing CORS restrictions on AEM (which could be a hassle if we ask customers to do so)
- and for providing a publicly usable url for the asset

I logged  #26 to see if we could eliminate this proxy if the above 2 things can be fulfilled without it.

**Comment by mfrisbey** [comment]
I was able to add a "Copy" button _outside_ the asset selector that handles this as part of my latest [PoC](https://github.com/mfrisbey/franklin-tutorial/tree/d751f16abee8be093eb5f8a185983a96cf937f10). Having the button outside the selector gives us a lot more control over its behavior.

@sdmcraft In addition, the PoC above eliminates the need for the proxy; instead it uses the repository API's `download` rel, which can provide a pre-signed URL for a rendition's binary. The solution involves solving a CORS issue, which I've documented as #45.
